docs/how_to/kubernetes.md

```diff
@@ -0,0 +1,279 @@
+# M3DB on Kubernetes
+
+M3DB on Kubernetes is currently in the alpha phase of development. We currently provide static manifests to bootstrap a
+cluster. In the future we hope to create an [operator](https://coreos.com/operators/) and leverage [custom resource
+definitions](https://v1-10.docs.kubernetes.io/docs/concepts/api-extension/custom-resources/) (CRDs) to automatically
+handle operations such as managing cluster topology, but for now our manifests should be adequate to get started.
+
+## Prerequisites
+
+M3DB is designed to run on SSDs. Every incoming write is buffered to a commit log, and because of this it performs
```

Comment:
nit: Could you avoid implying users have to use SSDs, couch the "prerequisite" as a suggestion not a must-follow?